Snippets/ASB/ASB_7/DeadLetterQueueForwarding.cs

```diff
@@ -0,0 +1,15 @@
+using NServiceBus;
+
+class DeadLetterQueueForwarding
+{
+    void ConfigureDeadLetterQueueForwarding(EndpointConfiguration endpointConfiguration)
+    {
+        #region asb-configure-dead-letter-queue-forwarding
+
+        var transport = endpointConfiguration.UseTransport<AzureServiceBusTransport>();
+        transport.Queues().ForwardDeadLetteredMessagesTo(queuename => queuename != "error" && queuename != "audit" && queuename != "centralizeddlq", "centralizeddlq");
+        transport.Subscriptions().ForwardDeadLetteredMessagesTo("centralizeddlq");
```

Comment:
So, the DLQ has to be setup individually, i.e. Queues() vs Subscriptions() ? What's the benefit of doing that as opposed to just on the endpoint level once?